commands/course.js

```diff
@@ -1,41 +1,69 @@
-const Discord = require('discord.js');
-const fetch = require('node-fetch');
+const Discord = require("discord.js");
+const { request, gql } = require("graphql-request");
+const { uwflowQuery, uwflowEndpoint } = require("../util/uwflow-constants");
+
+const courseMatcher = /([a-zA-Z]{2,}[ ]?\d+[a-zA-Z]?)/g;
```

Comment:
I believe that the courseMatcher regex is declared but never used; is this intentional?

Reply:
See lines 40-43:
```javascript
const prereqs =
    course.prereqs?.replaceAll(courseMatcher, replacer) ?? "None";
const antireqs =
    course.antireqs?.replaceAll(courseMatcher, replacer) ?? "None";
```

Reply:
My bad, I think I was looking at the wrong commit. Thanks.